apps/condo/domains/miniapp/schema/MessageAppBlackList.js

```diff
@@ -10,7 +10,7 @@ const { getMessageTypeField } = require('@condo/domains/notification/schema/fiel
 
 
 const MessageAppBlackList = new GQLListSchema('MessageAppBlackList', {
-    schemaDoc: 'Rule for blocking specific type of messages for apps',
+    schemaDoc: '[Deprecated] Rule for blocking specific type of messages for apps',
```

Comment:
I suppose you should note new one in the docs like that - 
```suggestion
    schemaDoc: '[Deprecated] Rule for blocking specific type of messages for apps. Use B2CAppMessageSetting',
```